Common.Configurations.targets

```diff
@@ -34,7 +34,7 @@
 
   <ItemGroup Condition="$(AssemblyName.Contains('.UnitTests')) != 'true'">
 	  <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.33" />
-	  <PackageReference Include="LT.DigitalOffice.Models" Version="1.0.24" />
+	  <PackageReference Include="LT.DigitalOffice.Models" Version="1.0.26" />
```

Comment:
why 26 if were 24?

Reply:
I just updated it a little, this is a library with our models, it was not required to update it in pr, but it will not make it worse